.travis.yml

```diff
@@ -40,6 +40,12 @@ env:
   - NPM_TAG=latest IMAGE_NAME=theia-https NODE_VERSION=10 ENV_VARS="-e token=" PORT=10443
   - NPM_TAG=next IMAGE_NAME=theia-https NODE_VERSION=10 ENV_VARS="-e token=" PORT=10443
 
+before_install:
+- nvm install 10
+- nvm use 10
+- npm install -g eclint
+- eclint check "**/*" | echo "Warning! Formatting errors encountered..."
```

Comment:
I think we need to use the "double pipe"  (`||`) so it's interpreted as a logical "or" operation rather then a pipe, else it will output the "echo", [whether the check failed or not](https://travis-ci.org/theia-ide/theia-apps/jobs/618601791#L209):

```
$> /bin/true | echo "allo"
allo
$> /bin/false | echo "allo"
allo
$> /bin/true || echo "allo"
$> /bin/false || echo "allo"
allo

```

Reply:
Yes, you're right!

Reply:
Yes, you're right!